src/network/network_survey.h

```diff
@@ -36,7 +36,11 @@ class NetworkSurveyHandler : public HTTPCallback {
 
 	constexpr static bool IsSurveyPossible()
 	{
+#ifdef SURVEY_KEY
```

Comment:
Normally if you don't set a key, it still sends surveys. These are processed in the backend as "unverified" (as the key was missing). Most useful to index versions that are not official. And during development to check if nothing breaks.

This would lose that ability. Not saying it is a bad thing, just mentioning the reason why this is what it is.